### PR TITLE
Releases/v3

### PR DESCRIPTION
**Pull request by srdanrasic**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ inputs:
 
   tag-prefix:
     description: "Prefix that is used for the git tag"
-    default: "v"
```
Comment (TriPSs) [inline action.yml:56]: This would be a breaking change, can you revert?

File: action.yml [hunk 2]
```diff
@@ -140,6 +140,11 @@ inputs:
     description: "Adds the generated changelog as Action Summary"
     default: "false"
     required: false
+    
+  breaking-bumps-minor:
```
Comment (TriPSs) [inline action.yml:144]: Would this logic not be better placed in a preset?

**Review by TriPSs — CHANGES_REQUESTED** [review]
Funny that for the actions to generate a changelog based off conventional commits you did not use a single conventional commit, If I where to merge this it would not show up in the changelog.

Please answer/process the comments and commit them **with** a commit following the guidelines.

**Comment by srdanrasic** [comment]
Sorry @TriPSs, this was opened completely accidentally.

**Pull request closed without merging**